src/web/app/page.component.html

```diff
@@ -1,9 +1,9 @@
 <nav class="navbar navbar-expand-lg navbar-dark fixed-top">
   <a routerLink="/web">
-    <img src="assets/images/teammateslogo.png" alt="TEAMMATES [Logo]">
+  <img src="assets/images/teammateslogo.png" alt="TEAMMATES [Logo]">
   </a>
   <button class="navbar-toggler d-lg-none" type="button" (click)="isCollapsed = !isCollapsed" [attr.aria-expanded]="!isCollapsed" aria-label="Toggle navigation">
-    <span class="navbar-toggler-icon"></span>
+  <span class="navbar-toggler-icon"></span>
```

Comment:
Try not to have this kind of unrelated changes.

Reply:
okay, I think my auto-prettify did that (will make sure it doesnt do that in future)